### PR TITLE
TES5: AVIF updates texture filename and skill category enum

### PR DESCRIPTION
**Pull request by jbostrus**

### CODE REVIEW

File: Core/wbDefinitionsTES5.pas [hunk 1]
```diff
@@ -8442,8 +8442,14 @@ procedure DefineTES5;
     wbEDID,
     wbFULL,
     wbDESCReq,
+    wbString(ICON, 'Image FileName', 0, cpNormal, True),
```
Comment (Infernio) [inline Core/wbDefinitionsTES5.pas]: is aRequired = True correct here? I can't find any AVIFs in Skyrim that have an ICON subrecord.

**Review by Infernio — APPROVED** [review]

File: Core/wbDefinitionsTES5.pas (hunk 1 above)
Comment (robertgk2017) [reply Core/wbDefinitionsTES5.pas]: image
It's not set as required?   You're looking at the first commit, not the changes after his 2nd commit.  Go by the Files Changed tab, not individual commits.

Comment (Infernio) [reply Core/wbDefinitionsTES5.pas]: Jonathan fixed it after I left that comment. My comment was yesterday, the second commit was 7 hours ago.

Comment (robertgk2017) [reply Core/wbDefinitionsTES5.pas]: Oh i was reading the wrong timestamp, ignore me.  Your comment only showed up just today from the notification, so was confused

Comment (jbostrus) [reply Core/wbDefinitionsTES5.pas]: Heh I never even got the alert about the comment. I just noticed I goofed that up when I was looking at something else and was like..."Why is ICON here on a empty record?"

**Pull request merged**